Tests/DatadogTests/Datadog/Mocks/DatadogInternal/UploadMock.swift

```diff
@@ -42,21 +41,21 @@ internal struct FailingRequestBuilderMock: FeatureRequestBuilder {
 }
 
 extension URLRequestBuilder.QueryItem: RandomMockable, AnyMockable {
-    static func mockRandom() -> Self {
+    public static func mockRandom() -> Self {
```

Comment:
To not make all of them public, we can make the `TestUtilities` import `@testable`.

If you guys have any preference, happy to change that :)

Reply:
I think it might make sense to do `@testable import` to limit the number of code changes in this PR. Otherwise we'll be running into git conflicts quite heavily.